src/Auth/Controller/AuthController.php

```diff
@@ -18,15 +18,14 @@
 
 final class AuthController
 {
-    private WebControllerService $webService;
     private ViewRenderer $viewRenderer;
-    private AuthService $authService;
 
-    public function __construct(ViewRenderer $viewRenderer, AuthService $authService, WebControllerService $webService)
-    {
+    public function __construct(
+        private AuthService $authService,
+        private WebControllerService $webService,
+        ViewRenderer $viewRenderer,
```

Comment:
```suggestion
        private ViewRenderer $viewRenderer,
```

Reply:
This will not allow you to specify `readonly` for `AuthController::viewRenderer` in future. OK ?